src/app/parcels/batch/FieldValidationFunctions.ts

```diff
@@ -0,0 +1,11 @@
+import { GridEditCellProps, GridPreProcessEditCellProps } from "@mui/x-data-grid";
+import { phoneNumberRegex } from "@/common/format";
+
+export const phoneNumberValidation = (params: GridPreProcessEditCellProps): GridEditCellProps => {
+    const unformattedInput = params.props.value.replaceAll(/[\s-()]/g, "");
+    const hasError = unformattedInput.match(phoneNumberRegex) === null;
+    return {
+        ...params.props,
+        error: hasError,
+    };
+};
```

Comment:
Can we commonise this with the phone number validators elsewhere?

Reply:
Its using the same exported regex from the form validation, is this what you mean?